aea/protocols/generator.py

```diff
@@ -1185,8 +1182,8 @@ def _custom_types_module_str(self) -> str:
             cls_str += '        """\n'
             cls_str += "        raise NotImplementedError\n\n"
 
-            cls_str += "    @classmethod\n"
-            cls_str += '    def decode(cls, {}_protobuf_object) -> "{}":\n'.format(
+            cls_str += "    @staticmethod\n"
```

Comment:
Decode is a classmethod, only encode is not :)

Reply:
Done